plumbing/format/gitattributes/attributes.go

```diff
@@ -0,0 +1,214 @@
+package gitattributes
```

Comment:
Maybe it better simply call the package `attributes`

Reply:
I did wonder that, but stuck with `gitattributes` as go-git already has `gitignore`. Git's own documentation also refers to it as "gitattributes" (https://git-scm.com/docs/gitattributes).

I don't mind either way though